mediator/med_phases_prep_atm_mod.F90

```diff
@@ -4,6 +4,31 @@ module med_phases_prep_atm_mod
   ! Mediator phases for preparing atm export from mediator
   !-----------------------------------------------------------------------------
 
+  use med_kind_mod          , only : CX=>SHR_KIND_CX, CS=>SHR_KIND_CS, CL=>SHR_KIND_CL, R8=>SHR_KIND_R8
```

Comment:
why was this done?  It's better inside the subroutine.

Reply:
Actually I disagree. For the kind - it should be a module level use statement since it could apply to any subroutine that uses it.